File: src/app/core/dtos/customers/customerOutput.ts
import { customerTypesOptions } from 'src/app/shared/support/enums/customer-types-options';

export class CustomerOutput {
  public id!: string;
  public name!: string;

  public type!: customerTypesOptions;

  public initialBalance!: number;

  public actualBalance!: number;

  public isArchived!: boolean;
}

File: backend/finan-exp-backend/Customers/Dtos/CustomerOutput.cs
using Customers.Enums;
using Customers.Models;

namespace Customers.Dtos
{
  public class CustomerOutput : ICustomerModel
  {
    public Guid Id { get; set; }
    public string Name { get; set; }

    public CustomersTypeOptions Type { get; set; }

    public decimal InitialBalance { get; set; }

    public decimal ActualBalance { get; set; }

    public bool IsArchived { get; set; }
    public decimal TransferValue { get; set; }
  }
}
